supplier/spring-jdbc-supplier/src/main/java/org/springframework/cloud/fn/supplier/jdbc/JdbcSupplierConfiguration.java

```diff
@@ -57,7 +56,7 @@ public JdbcSupplierConfiguration(JdbcSupplierProperties properties, DataSource d
 	}
 
 	@Bean
-	public MessageSource<Object> jdbcMessageSource(
+	public JdbcPollingChannelAdapter jdbcMessageSource(
```

Comment:
The change makes sense (use concrete type) but just wondering if it was necessary for part of this change or just an improvement along the way?

Reply:
That is just a side effect changes in this class.
According to SF rules, the type of the bean must be as specific as possible.
Therefore not a plain `MessageSource` interface, but really what we produce here - `JdbcPollingChannelAdapter`.

Reply:
That is what I figured. Thanks for clarification.